views/timelisting.tmpl

```diff
@@ -17,27 +17,32 @@
                 {{$pending := (subtract $lastGrouping .Offset)}}
                 {{$dropVal := $lastGrouping}}
                 {{if gt $lastGrouping 200}}{{$dropVal = 200}}{{end}}
-                <span class="h2 d-flex pt-4 pb-1 pr-2">{{.TimeGrouping}}</span>
+                <span class="h2 d-flex pt-4 pb-1 pr-2">
+                    {{.TimeGrouping}}
+                    <span class="dcricon-info fs24 ml-3 mt-2"></span>
+                </span>
 
                 <div class="pb-1 d-flex justify-content-end align-items-end flex-wrap">
                   <span class="fs12 nowrap text-secondary px-2 my-2">
                       {{intComma (add .Offset 1)}} &ndash; {{intComma $oldest}} of {{ intComma $lastGrouping }} rows
                   </span>
-                  <span class="fs12 nowrap text-right">
+                  <span class="fs12 nowrap text-right mr-2">
                       <select
                           data-target="pagenavigation.pagesize"
                           data-action="change->pagenavigation#setPageSize"
                           data-offset="{{$.Offset}}"
                           data-offsetkey="offset"
                           class="dropdown text-secondary my-2 {{if lt $pending 10}}disabled{{end}}"
-                          {{if lt $pending 10}}disabled="disabled"{{end}}
+                          {{if lt $pending 10}}disabled='disabled'{{end}}
```

Comment:
Why the single quotes?

Reply:
bringing them back...must have been accidental